docs-new/docs/contribute/software/dev-environ.md

```diff
@@ -144,27 +145,43 @@ Tests are run on your local machine.
 Study the detailed log that is produced to identify why the test failed.
 Study these errors, modify your code, and rerun the test until it passes.
 
-1. Use your IDE to run unit tests on your code.
+### Unit tests
 
-1. Run the integration tests from the root directory of your clone:
+Use your IDE to run unit tests on your code.
 
-```shell
-make integration-test-local
-```
+### Integration tests
 
-   `integration-test-local` cleans up after the test.
+Run the integration tests from the root directory of your clone:
 
-1. From the `lifecycle-operator` directory, run the component test:
+  ```shell
+  make integration-test-local
+  ```
 
-```shell
-make component-test
-```
+This runs a series of Kuttl
+([KUbernetes Test TooL](https://kuttl.dev/))
+tests locally and then cleans up your local environment.
+You can run individual tests with the `kubectl` command;
+see the *Makefile* for the specific syntax of each test.
 
-1. From the `lifecycle-operator` directory, run the end-to-end tests:
+Logs are printed only when the test fails.
+To force a failure,
+modify some value that appears in an assert file of the test.
```

Comment:
This information is not accurate. Logs are always printed no matter what the test outcome is.
I think we can just remove this whole paragraph.

Reply:
actually, test logs are always printed and the pod logs are only printed when tests fail to be exact

Reply:
It is removed